Assets/Scripts/Controllers/TradeController.cs

```diff
@@ -63,6 +64,8 @@ public void CallTradeShipTest(Furniture landingPad)
         SpriteRenderer spriteRenderer = go.AddComponent<SpriteRenderer>();
         spriteRenderer.sprite = SpriteManager.GetSprite("Trader", "BasicHaulShip");
         spriteRenderer.sortingLayerName = "TradeShip";
+        controller.Animation = new SpritenameAnimation("flying", new string[] { "BasicHaulShip_01", "BasicHaulShip_02" }, 1 / 4);
```

Comment:
this shouldn't be a static string array, but is dependent of the trader prototype